src/test/java/gov/healthit/chpl/aqa/asserts/DpmanagementUploadPageAsserts.java

```diff
@@ -36,4 +55,16 @@ public void verifyLongTestTaskIdAndParticipantIdErrorsOnUpload(final String erro
         assertTrue(actualString.contains(errorMessage), "Expect \"" + errorMessage + "\" to be found in \"" + actualString + "\"");
     }
 
+    /**
+     * Load listing details to verify surveillance was uploaded successfully.
+     */
+    @Then("^I see that surveillance was uploaded successfully to CHPL and listing details show surveillance activity$")
+    public void verifySurveillanceConfirmWasSuccessful() {
+        WebElement accordion = ListingDetailsPage.surveillanceActivitiesAccordion(getDriver());
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", accordion);
```

Comment:
I don't know where to mention this, but if you go to the regular details page, but put "/surveillance" at the end of it, it'll open with that accordion open already: https://chpl.ahrqdev.org/#/listing/9672/surveillance

Reply:
I'll try this